deploy/kubernetes/dolphinscheduler/values.yaml

```diff
@@ -794,6 +794,97 @@ alert:
       labels: {}
       # -- serviceMonitor.annotations ServiceMonitor annotations
       annotations: {}
+  configmap:
```

Comment:
I don't think we need to put everything into values. In the future, probably we will have separated configmaps for api server, master server and worker server. Why not put them directly in `configmap-dolphinscheduler-alert.yam`?

Reply:
> I don't think we need to put everything into values. In the future, probably we will have separated configmaps for api server, master server and worker server. Why not put them directly in `configmap-dolphinscheduler-alert.yam`?

it seems like 2 points?
1.not put everything into values, i agree. maybe we can choose some important items here.
2.because we can not just put application.yaml into cm but other config items. i think it may be more extendable if configured in values.yaml

Reply:
We could put application.yaml into configmap, and make some important items of application.yaml as variables and access them through `values.yaml`. For example api server, alert server and master server both need access to db and there are db related configs in the `application.yaml` for all of them. So you may put db related configs in values.yaml, and share those configs across different `application.yaml` for alert server, api server and master server. The same for registry related configs.

Reply:
> We could put application.yaml into configmap, and make some important items of application.yaml as variables and access them through `values.yaml`. For example api server, alert server and master server all need access to db and there are db related configs in the `application.yaml` for all of them. So you may put db related configs in values.yaml, and share those configs across different `application.yaml` for alert server, api server and master server. The same for registry related configs.

I got your point.
will apply these changes.